app/containers/ModalStack.js

```diff
@@ -39,16 +46,36 @@ ModalContent.propTypes = {
   closeModal: PropTypes.func.isRequired
 }
 
-const Modal = ({ modals, closeModal }) =>
-  modals.map(modal => {
-    return (
-      <ModalOverlay onClose={() => closeModal(modal.id)} key={modal.id}>
-        <ModalContent type={modal.type} closeModal={closeModal} />
-      </ModalOverlay>
-    )
-  })
+function ModalStack(props) {
+  const { modals, closeModal } = props
+
+  const closeLast = () => modals && modals.length && closeModal(modals[modals.length - 1].id)
```

Comment:
Calling `closeModal` without an argument causes it to push the last one off the stack.

Reply:
updated